ilan/responsefile.txt

```diff
@@ -0,0 +1,17 @@
+configType=remote
```

Comment:
Docs don't say anything about this file... Maybe this usage information should in a separate directory anyway as it is common to both image?